packages/core/.gitignore

```diff
@@ -58,3 +58,4 @@ utils/nanoid.d.ts
 index.cjs
 index.js
 index.d.ts
+yarn.lock
```

Comment:
check if the `yarn.lock` is actually get ignored. I can see the new updates of the file is pushed.